CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ### CHANGE LOG
 
 =======
+### v2.4.12
+> Point website searches to /search/ instead of /apachesolr_search/
```

Comment:
It is quite minor, but I think it is probably better use either "points" or "pointed" just to keep consistent with the verb tense?